src/component/parser/json/converter/ProcessConverter.ts

```diff
@@ -68,10 +66,9 @@ interface EventDefinition {
   counter: number;
 }
 
-@JsonConverter
-export default class ProcessConverter extends AbstractConverter<Process> {
+export default class ProcessConverter extends AbstractConverter<void> {
```

Comment:
:question: I thought that we would have been able to remove global 'converted' arrays and encapsulate them in a dedicated processing short life instance to avoid any side effect/concurrent accesses?
Is this plan in another PR?

Reply:
I didn't think about that. 
I don't really see how to do that. 
If you want, do it. Or we can do it in another PR. As you wish ^^

Reply:
so in another PR, later :smiley_cat: